deps/openssl.mk

```diff
@@ -34,6 +34,8 @@ else ifeq ($(ARCH),ppc64le)
 OPENSSL_TARGET := linux-ppc64le
 else ifeq ($(ARCH),powerpc64le)
 OPENSSL_TARGET := linux-ppc64le
+else ifeq ($(ARCH),riscv64)
+OPENSSL_TARGET := linux64-riscv64
```

Comment:
Reference for this target name: https://github.com/openssl/openssl/blob/817a2b2b4955da0233fe7e6e4bd16c0255262b4f/Configurations/10-main.conf#L819.  Or run `./Configure --help` on a riscv64-linux box.